### ISSUE TITLE
MD046 false-positive in presence of indented HTML comment

### ISSUE THREAD

**Issue by senekor** [rvben/rumdl#576]
### rumdl version

0.1.72

### Installation method

Other

### Bug description

rumdl installation method: nix.

Rumdl reports a warning when there is only one fenced code block after an HTML comment with indented content. I assume the indented comment is accidentally flagged as a code block. Then, rumdl thinks that the predominant code block style in this document is idented. Therefore, it flags the fenced code block as an inconsistency.

### Steps to reproduce

````md
# MD046 false-positive reproduction

<!--
    This is just an indented comment, not a code block.

    A second line is required to trigger the false-positive.

    Actually, three lines are required.
-->

```md
This should be fine, since it's the only code block and therefore consistent.
However, rumdl emits a warning saying "Use indented code blocks".
```
````

### Expected behavior

No error reported, since there is only one (and therefore consistent) code block.

### Actual behavior

Rumdl emits a warning saying "Use indented code blocks".

### Configuration

I don't think it should be relevant, but for completeness, here's my global config:

```toml
# https://rumdl.dev/global-settings/

[global]
disable = [
    # Keep lines short for better readability
    # https://rumdl.dev/md013/
    # This interferes with the "one sentence per line" style.
    "MD013",
    # No HTML tags
    # https://rumdl.dev/md033/
    # Inline HTML is just too useful for better layout / formatting.
    "MD033",
    # Code blocks should have a language specified
    # https://rumdl.dev/md040/
    # No language meaning text is pretty obvious.
    "MD040",
    # Keep italic / bold text formatting consistent
    # https://rumdl.dev/md049/ https://rumdl.dev/md050/
    # Having this "fix" in a commit is worse than the original offense.
    # These rules could be enabled if a project enforces `rumdl check` in CI.
    "MD049",
    "MD050",
]

# Use consistent numbers for ordered lists
# https://rumdl.dev/md029/
[MD029]
# Style "one" leads to less churn when elements are added, removed or reordered.
style = "one"
```

### Operating system

Linux

**Comment by rvben** [comment]
Fixed in [v0.1.78](https://github.com/rvben/rumdl/releases/tag/v0.1.78). Thanks for the report, @senekor!
